lib/Button/index.js

```diff
@@ -1,29 +1,32 @@
 import React, { Component, PropTypes } from 'react';
 
-// Ideally, we'd just be doing this, but we need to account for Sass
-// mixins and settings... to study a bit more.
-// require('./styles.scss');
+/**
+ * Example Usage
+ *
+ * <Button
+ *  types={['clear', 'collapsed']}
+ *  clickHandler={() => {}}
+ * >
+ *   ...test goes here
+ * </Button>
+ */
 
 class Button extends Component {
-  static getClassName(type) {
-    switch (type) {
-      case 'primary':
-        return 'btn btn--primary';
-      case 'secondary':
-        return 'btn btn--secondary';
-      case 'danger':
-        return 'btn btn--warning';
-      case 'link':
-        return 'btn btn--link';
-      case 'clear':
-        return 'btn btn--clear';
-      case 'clear-and-collapsed':
-        return 'btn btn--clear btn--collapsed';
-      case 'neutral':
-      default:
-        return 'btn btn-default--light';
-    }
-  }
+  static defaultProps = {
```

Comment:
Switched to this placement for default Props and prop validation. I felt like I was missing out on knowledge about the component by keeping these at the bottom of the file.

Moving it up like this means that I digest the contents of this component a little better. What do you think @magalhini?